pkg/apis/policy/v1alpha1/override_types.go

```diff
@@ -100,6 +102,31 @@ type Overriders struct {
 	// ArgsOverrider represents the rules dedicated to handling container args
 	// +optional
 	ArgsOverrider []CommandArgsOverrider `json:"argsOverrider,omitempty"`
+
+	// LabelsOverrider represents the rules dedicated to handling workload labels
+	// +optional
+	LabelsOverrider []CommandKeyValuePairOverrider `json:"labelsOverrider,omitempty"`
+
+	// AnnotationsOverrider represents the rules dedicated to handling workload annotations
+	// +optional
+	AnnotationsOverrider []CommandKeyValuePairOverrider `json:"annotationsOverrider,omitempty"`
+}
+
+// CommandKeyValuePairOverrider represents the rules dedicated to handling workload labels/annotations
+type CommandKeyValuePairOverrider struct {
```

Comment:
I didn't get the name `Command` here. I tend to names like `LabelAnnotationOverrider`.

Reply:
😂，spelling mistake . I want to express `CommonKeyValuePairOverrider `

Reply:
So, what do you think about `LabelAnnotationOverrider`?

Reply:
Agree with you.